tdp/cli/commands/plan/__init__.py

```diff
@@ -0,0 +1,27 @@
+import click
+from dotenv import load_dotenv
+
+from tdp.cli.utils import env
+
+from .dag import dag
+from .reconfigure import reconfigure
+from .resume import resume
+from .run import run
+
+CONTEXT_SETTINGS = dict(help_option_names=["-h", "--help"])
+
+
+@click.group(
+    context_settings=CONTEXT_SETTINGS,
+    invoke_without_command=True,
+    short_help="Generate a deployment plan.",
+)
+@env
```

Comment:
Why do you need to add `@env`. It is already done by the parent command no ?

Reply:
True, I though I had to put it again but that's not useful!